source/code/plugins/changetracking_lib.rb

```diff
@@ -50,6 +50,21 @@ def self.packageXMLtoHash(packageXML)
         ret
     end
 
+    def self.fileInventoryXMLtoHash(fileInventoryXML)
+        fileInventoryHash = instanceXMLtoHash(fileInventoryXML)
```

Comment:
If a field, used below, does not exist or is optional (eg, `Contents`), IMO It would be good to not populate it in the returning hash (`ret`).